tap_google_search_console/client.py

```diff
@@ -205,6 +218,16 @@ def check_sites_access(self):
             body = {'startDate': '2021-04-01', 'endDate': '2021-05-01'}
             self.post(path=path, data=json.dumps(body))
 
+    # during 'Timeout' error there is also possibility of 'ConnectionError',
+    # hence added backoff for 'ConnectionError' too.
+    @backoff.on_exception(backoff.expo,
```

Comment:
Added backoff for `get_access_token` here as this function is also called from the `request` function, and when there will be Timeout error then the backoff will happen 25 times rather than 5 times.